### PR TITLE
optimize code style 

### PR DESCRIPTION
**Pull request by slievrly**
## What is the purpose of the change

optimize code style :
1. Constant naming rules require letters and underscore combinations
2. All override methods need to add @override

## Brief changelog

optimize code style 

## Verifying this change

ci

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/ActiveLimitFilter.java [hunk 1]
```diff
@@ -1,120 +1,120 @@
...
-        private void notifyFinish(RpcStatus rpcStatus, int max) {
-            if (max > 0) {
-                synchronized (rpcStatus) {
-                    rpcStatus.notifyAll();
-                }
-            }
-        }
-    }
-}
+/*
```
Comment (kexianjun) [inline dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/ActiveLimitFilter.java]: Is there something wrong with this file?

File: dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java [hunk 2]
```diff
@@ -1,708 +1,708 @@
...
-        public void setSubscribeUrl(URL subscribeUrl) {
-            this.subscribeUrl = subscribeUrl;
-        }
-
-        public void setRegisterUrl(URL registerUrl) {
-            this.registerUrl = registerUrl;
-        }
-    }
-}
+/*
```
Comment (ralf0131) [inline dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java]: Please do not change the line ending.

**Comment by slievrly** [comment]
@ralf0131 why use CRLF line separators？

**Comment by slievrly** [comment]
@ralf0131 @kexianjun  already revert.

**Pull request merged**